packages/lib/cva/cva.test.ts

```diff
@@ -0,0 +1,52 @@
+import { applyStyleToMultipleVariants } from "./cva";
+
+describe("CVA Utils", () => {
```

Comment:
I wrote a small util to at least give us the option to use arrays for variants, which makes us have quite a bit less compound variant definitions 😄  Hoping my PR mentioned in the description of this PR gets merged soon, since the implementation in CVA itself is way simpler 🔥